android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -754,6 +758,9 @@ public void receiveCommand(WebView root, int commandId, @Nullable ReadableArray
       case COMMAND_CLEAR_HISTORY:
         root.clearHistory();
         break;
+      case COMMAND_RELEASE:
+        // no-op for now
```

Comment:
The iOS code in this PR is ready for review, and while that's getting reviewed, I'll see if I can get an Android implementation working.